lib/gateways/snapshot-gateway.js

```diff
@@ -88,11 +93,19 @@ class SnapshotGateway {
   async save({ snapshot }) {
     if (!snapshot) throw new ArgumentError('snapshot cannot be null.');
 
-    const updateExpression = ['set assets = :a', 'vulnerabilities = :v'];
+    // remove the dynamodb ttl on save
+    delete snapshot.ttl;
```

Comment:
Is there ever a chance that we could save an empty snapshot and end up removing the TTL but not having any content still? I wonder if check if the snapshot has data before doing this would be worthwhile?

Reply:
that's a good point actually